### PR TITLE
feat-added modal component

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added two types of modal
1)Simple Modal
2)Scrolling Modal

### CODE REVIEW

File: componets/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	overflow-y: scroll;
+}
+.scrolling__modal {
+	height: 12rem;
+}
+.modal__dismiss {
+	display: flex;
+	justify-content: space-between;
+}
+.modal__dismiss i {
```
Comment (KanekarSnehal) [inline componets/modal/modal.css:31]: give consistent space after each code block.

File: componets/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+							style="min-height: 280px"
```
Comment (KanekarSnehal) [inline componets/modal/modal.html:145]: Suggestion: avoid using inline styles, you can write this style in css file for all iframes

**Review by KanekarSnehal — COMMENTED** [review]
Great work.
Code looks good, I liked that you have followed good naming conventions.

File: componets/modal/modal.css (hunk 1 above)
Comment (prathmeshjagtap) [reply componets/modal/modal.css:31]: yes done

File: componets/modal/modal.html (hunk 2 above)
Comment (prathmeshjagtap) [reply componets/modal/modal.html:145]: yes i tried that but different iframes need different height so I am using inline styles only for  iframes

**Review by Richa-rani02 — APPROVED** [review]

**Review by KanekarSnehal — APPROVED** [review]

**Pull request merged**